pkgs/development/python-modules/socketio-client/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, requests
+, six
+, websocket-client
+, coverage
+, nose
+, fetchFromGitHub }:
+buildPythonPackage rec {
+  pname = "socketio-client";
+  version = "0.7.2";
+
+  src = fetchFromGitHub {
+    owner = "invisibleroads";
+    repo = "socketio-client";
+    rev = version;
+    sha256 = "sha256-71sjiGJDDYElPGUNCH1HaVdvgMt8KeD/kXVDpF615ho=";
+  };
+
+  propagatedBuildInputs = [
+    six
+    websocket-client
+    requests
+  ];
+
+  checkInputs = [
+    coverage
+    nose
+  ];
+
+  # Perform networking tests.
+  doCheck = false;
+
+  passthru.updateScript = ./update.sh;
```

Comment:
Please remove this. Doesn't add value as the module is not updated on a regular base (last action in the repo was over 4 years ago) and Python modules are updated in bulk.